docs/source/basic_tutorials/non_core_models.md

```diff
@@ -0,0 +1,17 @@
+# Non-core Model Serving
+
+TGI supports various LLM architectures (see full list [here](../supported_models)). If you wish to serve a model that is not one of the supported models, TGI will fallback to the `transformers` implementation of that model. This means you will be unable to use some of the features introduced by TGI, such as tensor-parallel sharding or flash attention. However, you can still get many benefits of TGI, such as continuous batching or streaming outputs.
```

Comment:
```suggestion
TGI supports various LLM architectures (see full list [here](../supported_models)). If you wish to serve a model that is not one of the supported models, TGI will fallback to the `transformers` implementation of that model. This means you will be unable to use some of the features introduced by TGI, such as tensor-parallel sharding or flash attention. However, you can still get many benefits from TGI, such as continuous batching or streaming outputs.
```

Question: is TGI's flash attention coming from BetterTransformer? (Just curious, haven't looked at the code yet). In theory, people could still other optimizations like SDPA from PyTorch 2.0.1, if their models use them, right?

Reply:
I didn't benchmark them, we were thinking of it as nice to have

Reply:
I didn't benchmark them, we were thinking of it as nice to have. pinging @Narsil

Reply:
flash attention comes from flash attention tridao's repo.